config/crd/bases/druid.gardener.cloud_etcds.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.5.0
+    controller-gen.kubebuilder.io/version: v0.4.1
```

Comment:
Just a clarification. Is this an intended change? 
I imagine this happened when running `make test` locally before pushing the changes and hence is not intended.

Reply:
I don't know how this change happened!!

Reply:
I think it was done automatically. I didn't make any changes here. @amshuman-kr , Any Idea?

Reply:
I've seen this before
From what I gather, if the `controller-gen` version installed locally is of a different version, it updated this file to it's version before running `make test`
Why it tries to have the entire description in one line I'm not too sure though

Reply:
Good catch @aaronfern! I think if you update your `controller-gen` and `make generate` again, it would go away.

Reply:
However, I tried to freshly install `controller-gen` from the currently vendored package, ran `make manifests` and got exactly the same as I see in this PR. So, at least it is consistent with the vendoring. 

I think in the previous update the controller-gen was locally updated but the vendoring was not updated.

For consistent environment, we should ideally update `controller-gen` by updating vendoring first. Maybe we should pick this in a separate PR?